model/src/main/java/org/openremote/model/attribute/AttributeStateWithTimestamp.java

```diff
@@ -0,0 +1,20 @@
+package org.openremote.model.attribute;
+
+public class AttributeStateWithTimestamp extends AttributeState {
+    protected Long timestamp;
+
+    public AttributeStateWithTimestamp(){}
+
+    public AttributeStateWithTimestamp(AttributeState attributeState) {
+        this.ref = attributeState.ref;
+        this.value = attributeState.value;
+
+        this.timestamp = 0L;
```

Comment:
It's used [here](https://github.com/openremote/openremote/blob/a5e3bd96de67039c822bed471c0ab7208a261114/manager/src/main/java/org/openremote/manager/asset/AssetResourceImpl.java#L453), so that I can call the timestamped method and avoid duplicating code. Is there a better way to do that?